src/components/Span/SpanDetailsPanel.tsx

```diff
@@ -258,94 +261,126 @@ export const SpanDetailPanel = ({
   }, [result, debouncedSearch]);
 
   return (
-    <div className="z-10">
-      <div className="overflow-hidden text-sm">
-        <h2 className="uppercase">Span Details</h2>
-        <input type="text" placeholder="Search" value={search} onChange={(e) => setSearch(e.target.value)} />
-        <table className="w-full">
+    <div className="z-10 overflow-hidden text-sm">
+      <div className="px-2 py-2">
+        <div className="flex flex-col gap-4 items-start justify-between py-4 px-2 mx-4">
+          <div className="flex items-center justify-between gap-2 w-full">
+            <span className="uppercase text-lg font-light">Span Details</span>
+            <IconButton
+              size="xxl"
+              name="times"
+              variant="secondary"
+              aria-label="Close"
+              onClick={onClose}
+              className="w-4 h-4"
+            />
+          </div>
+          <Input
+            className="w-full"
+            type="text"
+            placeholder="Search details"
+            value={search}
+            onChange={(e) => setSearch(e.currentTarget.value)}
+            suffix={
+              search ? (
+                <IconButton name="times" variant="secondary" aria-label="Clear" onClick={() => setSearch('')} />
+              ) : (
+                <IconButton name="search" variant="secondary" aria-label="Search" />
+              )
+            }
+          />
+        </div>
+      </div>
+      <div className="px-4">
```

Comment:
Ensure this align better with the accordeon sections:

<img width="416" height="430" alt="Image" src="https://github.com/user-attachments/assets/1f5e5982-054e-4b38-8cfd-22a8d09f7adb" />

The scrollbar is over the copy button.